sw/device/tests/hmac_endianness_test.c

```diff
@@ -161,5 +222,27 @@ bool test_main(void) {
                       kHmacconfig_bigd_littlem,
                       &kExpectedShaDigest_shortmessage_bigendian);
 
+  LOG_INFO(
+      "Running test HMAC pass little-endian digest and little-endian "
+      "message with key_swap set to 0...");
+  run_test_hmacmode_endianness(
+      &hmac, kData, sizeof(kData), (uint8_t *)(&kHmacKey[0]),
+      kDifHmacEndiannessBig, kHmacconfig_littled_littlem, &kExpectedHmacDigest);
+
+  LOG_INFO(
+      "Running test HMAC pass little-endian digest and little-endian "
+      "message with key_swap set to 1 and with key swapped..");
+  run_test_hmacmode_endianness(
+      &hmac, kData, sizeof(kData), (uint8_t *)(&kHmacKey_swapped[0]),
+      kDifHmacEndiannessLittle, kHmacconfig_littled_littlem,
```

Comment:
```suggestion
      kKeyswapSetLittleendian, kHmacconfig_littled_littlem,
```
This is the key endian argument, right? I suggest using the enums from above.

Reply:
Done!